sharding-orchestration/sharding-orchestration-core/src/main/java/io/shardingsphere/orchestration/internal/registry/config/node/ConfigurationNode.java

```diff
@@ -108,4 +111,20 @@ private String getFullPath(final String schemaName, final String node) {
     private String getFullPath(final String node) {
         return Joiner.on("/").join("", name, ROOT, node);
     }
+    
+    /**
+     * Get schema Name.
+     * 
+     * @param configNodeFullPath config node full path
+     * @return schema name
+     */
+    public String getSchemaName(final String configNodeFullPath) {
+        String result = "";
+        Pattern pattern = Pattern.compile(getSchemaPath() + "/(\\w+)" + "(/datasource|/rule)?", Pattern.CASE_INSENSITIVE);
+        Matcher matcher = pattern.matcher(configNodeFullPath);
+        if (matcher.find()) {
+            result = matcher.group(1);
+        }
+        return result;
```

Comment:
If cannot match reg exp, and return "", what will happen?

Reply:
return IgnoredChangedEvent in createShardingOrchestrationEvent(final DataChangedEvent event)